### PR TITLE
added homepage

### PR DESCRIPTION
**Pull request by MurtazaBagwala07**
added folder structure

### CODE REVIEW

File: pages/homepage/homepage.css [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
...AVJADBlOmqsp5MOQ+6breDtI2pn1NRfkaUsNLK6q0qFg8mKjUj1a6T1zQiM623GxFs2Fs+m5A9dmzqUsRhqGR6gCpHkQZzbuNuldtbMSitWFIZTk3gejTVrzOvTiPMAeJ9QOnWmPStaJWihUrVURR0VVGgA9wkSlzD2gdnGXsu2x663vwNS1eQilu7TmeG4D5JH1uh/AQXlO3ZjfEKeLj7mrj+v3ScX36axsckDdPN+IvtB6GrxVatVexWVrS50Hdrpqw9fT1z93MocbV2YSjgDaOFqSjAfr0nX8RsIiJAREQNTt7YtWZXwWcmXXgsUekp/qPMSs9s7EvxG0tT0NfRtXUq3v8D6jLinixAwKsAykaEEAgj1ibODib4uXWP8Af7u0uK4Kmfn0t5/z5/r8VQbt4Xf5lFemqmzif9xfSP36ae+Ym8eb3+ZkW9VdyE/cA4V/AA++WTtTAxsCnJzKa1rtFLIvCSF4nIA0XoOenSVIBoJr8fxEZrRrpEKriMPuMcYpnczM2n9I/d+6QYnqpGdglas7noiKSx9wmg0/gsfdvanx7H4XOuXQoDedlZ6N7fA+v2zKmn3N3Tyqrq8q1u4C6/2fI2OpHNT4KPx9kl+Vs3is4kOitzb1Hz98qPavsy+bWbFX8XS0eflPy7/Dn2l2Hs3ib+6iM0antvy/339WDg4psfT6I5kyQgacp4opWtQqjl+JPmZ9JaezeArweLw9bT1n9o+EfrueW9PeXJ45IiJYsRERAREQEREBERAREQEREBPLqCCGAIIIII1BB6gieogQXbPZPsfKZnOMaHbq2NYah7QnNB90wsHsW2PW2rrlX89eG3I0X/4wsseIGHsvZlGLUKcaqumodErQIuvieXU+uZkRAREQEREBERAREQERED53UrYpR1V1PVWUMp9oM1N26mC51OMg/cL1j7lIm6iRMRPV5tStvzRE/Joqt0cBTqMcH96y1x9zMZtcTCqpHDTXVWPJEVR+AmRERER0K0rX8sRHpEQRESXoiIgIiICIiAiIgIiICIiAiIgIiICIiAiIgIiICIiAiIgIiICIiAiIgIiICIiAiIgIiICIiAiIgIiICIiAiIgIiICIiAiIgIiICIiAiIgIiICIiAiIgIiICIiAiIgIiICIiAiIgIiICIiAiIgIiICIiAiIgIiICIiAiIgIiICIiAiIgf/Z") ;
```
Comment (romabulani) [inline pages/homepage/homepage.css:32]: You can Remove the comments if not required

File: components/productCard/productCard.css [hunk 2]
```diff
@@ -0,0 +1,58 @@
+@import url("https://fonts.googleapis.com/icon?family=Material+Icons+Outlined%22");
+@import url("https://fonts.googleapis.com/icon?family=Material+Icons%22");
+
+*{
+    padding:0;
+    margin:0;
+    box-sizing: border-box;
```
Comment (romabulani) [inline components/productCard/productCard.css]: You can import reset.css instead of writing the code again

**Review by romabulani — APPROVED** [review]
Rest all looks good to merge

File: pages/homepage/homepage.css (hunk 1 above)
Comment (MurtazaBagwala07) [reply pages/homepage/homepage.css:32]: I will use a background image there but not been able to find the right image yet

File: components/productCard/productCard.css (hunk 2 above)
Comment (MurtazaBagwala07) [reply components/productCard/productCard.css]: yes will do that

**Review by Shah-Arsalan — APPROVED** [review]

**Pull request merged**